src/containers/Dev/DevOptions.tsx

```diff
@@ -4,15 +4,15 @@ import Snapshots from './Snapshots'
 import UserSelection from './UserSelection'
 
 const DevOptions: React.FC = () => {
-  return (
+  return process.env.NODE_ENV === 'production' ? null : (
```

Comment:
I have seen this today and agree:
https://github.com/openmsupply/application-manager-web-app/blob/ddbd13643747772d01f3b3f867b21059342fba82/src/containers/Application/ApplicationSummary.tsx#L14
I think we could do this now - not sure exactly how. Can we update the config as part of the building process to have the selected environment set in a field on `config.ts` file?

Reply:
Good idea, not sure how to do that though. You?

Reply:
I believe this was sorted in another PR